testing/trino-tests/src/test/java/io/trino/tests/TestLocalQueries.java

```diff
@@ -117,4 +117,17 @@ public void testTransformValuesInTry()
                         "FROM (VALUES map(ARRAY[1, 2], ARRAY[0, 0]),  map(ARRAY[28], ARRAY[2]), map(ARRAY[18], ARRAY[2]), map(ARRAY[4, 5], ARRAY[1, 0]),  map(ARRAY[12], ARRAY[3])) AS t(m)",
                 "VALUES NULL, '{\"28\":14}', '{\"18\":9}', NULL, '{\"12\":4}'");
     }
+
+    @Override
+    protected Object[][] largeInValuesCountData()
+    {
+        return new Object[][] {
```

Comment:
We also need to add a case where `IN` list is in subexpression, e.g: `x IS NULL or y IN (...)`